src/hw-cbmc/hw_cbmc_parse_options.cpp

```diff
@@ -84,11 +84,8 @@ int hw_cbmc_parse_optionst::doit()
       all_properties_verifier_with_trace_storaget<multi_path_symex_checkert>>(
       options, ui_message_handler, goto_model);
 
-  solver_factoryt sf{options, namespacet{goto_model.symbol_table},
-                     ui_message_handler, false};
-  auto my_solver = sf.get_solver();
-  prop_convt &prop_conv =
-      static_cast<prop_convt &>(*my_solver->decision_procedure_ptr);
+  // TODO : extend the interface of verifier to expose the prop_convt
```

Comment:
Instead of doing do_unwind_module and add_constraints here, they should probably be done inside an implementatipon of goto_verifier/checker.

Reply:
Yes, that sound more elegant. I've re-worded the todo description.